meshnet/exitnode/fw.go

```diff
@@ -211,7 +168,11 @@ type TrafficPeer struct {
 	LocalNetwork bool
 }
 
-func resetPeersTraffic(peers []TrafficPeer, commandFunc runCommandFunc) error {
+func resetPeersTraffic(peers []TrafficPeer, interfaceNames []string, commandFunc runCommandFunc) error {
+	if err := clearMasquerading(commandFunc); err != nil {
+		return fmt.Errorf("clearing masquerade rules: %w", err)
+	}
+
 	for _, peer := range peers {
 		err := modifyPeerTraffic(peer.IP, "-D", true, true, commandFunc)
```

Comment:
It's not directly related to this ticket, but as you are working here it might be the best moment to fix one more thing. I think there is an issue that when a peer is removed then their filtering rules will remain undeleted because the removed peer is simply not in the list anymore. On the other hand we cannot simply delete all rules with the `nordvpn` comment from the `FORWARD` chain, because there is the all blocking rule which we do not touch. What do you think? Of course this can be a separate issue as well if the fix is not trivial, there are probably enough changes in this PR already :D